components/raftstore-v2/src/worker/tablet_gc.rs

```diff
@@ -98,6 +100,10 @@ pub struct Runner<EK: KvEngine> {
     // region_id -> [(tablet_path, wait_for_persisted)].
     waiting_destroy_tasks: HashMap<u64, Vec<(PathBuf, u64)>>,
     pending_destroy_tasks: Vec<PathBuf>,
+
+    // An independent pool to run tasks that are time-consuming but doesn't take CPU resources,
+    // such as waiting for RocksDB compaction.
+    background_pool: Worker,
```

Comment:
It's weird to see a worker is referencing another worker. Probably should use future pool instead.